src/util/sharing_map.h

```diff
@@ -545,6 +561,20 @@ class sharing_mapt
     const innert &n,
     std::function<void(const key_type &k, const mapped_type &m)> f) const;
 
+  /// Add a delta item to the delta view if the value in the `container` (which
```

Comment:
I prefer `\p container` to make clear you are referring to a parameter (there's no actual difference in rendering at the moment, though).